extensions/lab/text.js

```diff
@@ -29,6 +31,22 @@
     'Pixel',
   ];
 
+  const EXTRA_FONTS = [
+    'Scratch',
+    'Arial',
+    'Arial Black',
+    'Helvetica Neue',
+    'Calibri',
+    'Garamond',
+    'Times New Roman',
+    'Courier New',
+    'Brush Script MT',
+    'Impact',
```

Comment:
a lot of these are not going to work on Chrome OS (~32% of users), macOS (~7% of users), or Linux (~5% of users)

Reply:
fonts have always been tricky for this reason

Reply:
Should they be removed?

Reply:
removing them for now